cluster-scope/base/clusterrolebindings/metallb/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - metallb-allow-privileged-scc.yaml
```

Comment:
@larsks all of these should be in their separate folders like so: 

cluster-scope/base/clusterrolebindings/metallb-allow-privileged
cluster-scope/base/clusterrolebindings/metallb-system-controller
cluster-scope/base/clusterrolebindings/metallb-system-speaker

With each folder having the actual manifest named `clusterrolebinding.yaml` and one kustomization.yaml per manifest. 

This should apply to the `clusterroles` added with this PR as well. The value of `<resource>` in `cluster-scope/base/clusterrolebindings/<resource>` should match the `metadata.name` as much as it can. 

This is how the rest of the resources are also structured. Hope that makes sense.